docs/sources/monitor/monitor-docker-containers.md

```diff
@@ -0,0 +1,233 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tutorials/scenarios/monitor-docker-containers/
+description: Learn how to use Grafana Alloy to monitor Docker containers
+menuTitle: Monitor Docker
+title: Monitor Docker containers with Grafana Alloy
+weight: 200
+---
+
+# Monitor Docker containers with {{% param "FULL_PRODUCT_NAME" %}}
+
+Docker containers provide statistics and logs.
+The `docker stats` and `docker logs` commands show the metrics and logs in a terminal as a fixed snapshot in time.
+You can use {{< param "PRODUCT_NAME" >}} to collect your metrics and logs, forward them to a Grafana stack, and create a Grafana dashboard to monitor your Docker container.
+
+The `alloy-scenarios` repository provides series of complete working examples of {{< param "PRODUCT_NAME" >}} deployments.
+You can clone the repository and use the example deployments to understand how {{< param "PRODUCT_NAME" >}} can collect, process, and export telemetry signals.
+
+In this example scenario, {{< param "PRODUCT_NAME" >}} collects the Docker container metrics and logs and forwards them to a Loki destination.
+
+## Before you begin
+
+This example requires:
+
+* Docker
+* Git
+
+{{< admonition type="note" >}}
+The `docker` commands require administrator privileges.
+{{< /admonition >}}
+
+## Clone the scenarios repository
```

Comment:
Why not just host this inside the Alloy repo, in an "examples" folder? Then we can also add a CI step to make sure the configs there are still valid, in case we add or remove parameters to component configs. 

Also, maintaining another repo adds more maintenance overhead. It need to be setup with the right permissions for branches, secrets, etc.

Reply:
Also open to this aswell, I am basing this on the barrier to entry and contributing new examples on the fly. When I was building Telegraf examples: https://github.com/InfluxCommunity/Telegraf-Community-Configs as long as you don't mind the examples folder getting to big with examples I can get them moved across